RealmSwift/Object.swift

```diff
@@ -277,9 +277,15 @@ open class Object: RLMObjectBase, ThreadConfined {
 
      - parameter object: The object to compare the receiver to.
      */
-    open override func isEqual(_ object: Any?) -> Bool {
+#if os(macOS)
```

Comment:
Yes, it's because of that horrible, horrible method + Swift's `override` annotation requirement.

I'd be alright with giving it a new name, as long as we're aware that doing so will break a bit of symmetry with `RLMObject` (not sure that is even an issue at this point).